config.go

```diff
@@ -304,6 +304,16 @@ func SkipPageIndex(skip bool) FileOption {
 	return fileOption(func(config *FileConfig) { config.SkipPageIndex = skip })
 }
 
+// SkipBloomFilters is a file configuration option which when set to true,
+// prevents automatically reading the bloom filters when opening a parquet file.
+// This is useful as an optimization when programs know that they will not need
+// to consume the bloom filters.
+//
+// Defaults to false.
+func SkipBloomFilters(skip bool) FileOption {
```

Comment:
Would you ever call this with `false` as the argument?

Reply:
I guess if the state is received from user input (e.g. via a command line argument), it's convenient to be able to pass a variable, otherwise the program needs to branch to decide whether to include the option in the option list etc...